### ISSUE TITLE
API Skills list with 'Status' pending 

### ISSUE THREAD

**Issue by rottitime** [i-dot-ai/ellandi#669]
Ticket: https://trello.com/c/R6yuT09C/169-pending-skills

Support for 'pending status' on all  list of Skills for all custom skill suggestions

---

## Pending attribute 

 As per https://trello.com/c/R6yuT09C/169-pending-skills 

All new skills added that are missing from the current skills list to show as `pending: true`.
If the skill already exists then set as `pending: false`

Also change pending status once approved

## All Skill list Endpoints to show pending status

For all endpoints that output list of skills e.g. `GET/POST /api/me/skills/`, `GET/POST /api/me/` etc, can you add a new status attribute
```lang-js
[
  {
    "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
    "name": "string",
    "level": "Beginner",
    "validated": true,
    "pending":true
  }
]
```

---

## List of 'pending' skills

New API endpoint which returns pending skills only

`GET /api/me/skills/?pending=true`

---

### Request 
**Endpoint**:  `POST /api/me/skill/review/`
**Payload**: 
```lang-js
[
  {
    id: string //id of skill
    approve: 'Yes' || 'No' || null 
    comment: string || null
  }
]
```
For example
```
[
    {
        "id": "id0",
        "name": "Skill 0",
        "approve": null,
        "comment": null
    },
    {
        "id": "id1",
        "name": "Skill 1",
        "approve": "No",
        "comment": "my text"
    },
    {
        "id": "id2",
        "name": "Skill 2",
        "approve": null,
        "comment": null
    },
    {
        "id": "id3",
        "name": "Skill 3",
        "approve": "Yes",
        "comment": null
    },
    {
        "id": "id4",
        "name": "Skill 4",
        "approve": null,
        "comment": null
    }
]
```

### Response
status 200

**Comment by rottitime** [comment]
@nmenezes0 @Singletoned  what is the `"validated": true` that is currently returned? Is this to indicate if a status is pending?

**Comment by rottitime** [comment]
@nmenezes0 closing for now as the ticket as PO team have pushed this ticket back for review

**Comment by rottitime** [comment]
@Singletoned @nmenezes0 reopened after speaking to product owners

**Comment by rottitime** [comment]
@nmenezes0 @Singletoned Please notice the change that `/api/me/skill/review`. Payload has changed to array of values now

**Comment by nmenezes0** [comment]
Also fixes #743 (which duplicates some of this ticket).
